projects/cmake/build.sh

```diff
@@ -31,6 +32,7 @@ while echo $1 | grep ^- > /dev/null; do
 -help           <true|false>    : update the help database and build the YAML help generator. Defaults to false.
 -boost_root     string          : specify directory containing Boost headers (e.g. `/usr/include`). Defaults to unset.
 -boost_lib      string          : specify directory containing Boost libraries. (e.g. `/usr/lib`). Defaults to unset.
+-boost_include  string          : specify directory containing Boost includes. (e.g. `/usr/include`). Defaults to unset.
 -static_boost	<true|false>    : link using static Boost libraries. Defaults to false.
```

Comment:
I think it would also help to have `-debug_boost   <true|false>` or `-boost_debug   <true|false>`.  This could add the cmake argument `-DBoost_DEBUG=ON` and maybe `-DBoost_VERBOSE=ON`.

Reply:
Yep good idea, that would be easy to add. Will get on that tomorrow.